Automattic/vip-go-mu-plugins
# Update/jetpack9.0

## Description

Updates jetpack 9.0-beta to 9.0

## Checklist

Please make sure the items below have been covered before requesting a review:

- [ x ] This change works and has been tested locally (or has an appropriate fallback).
- [ x ] This change has relevant unit tests (if applicable).
- [ x ] This change has relevant documentation additions / updates (if applicable).

## Steps to Test

1. Check out PR.
1. Set `define( 'VIP_JETPACK_PINNED_VERSION', '9.0' );`
1. Go to `wp-admin` > `Jetpack``
1. should show 9.0 version in the footer.

Also, nothing else is broken (I don't know how to test everything, really:))
